### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by nxtn**

### CODE REVIEW

File: docs/standard/microservices-architecture/net-core-net-framework-containers/net-framework-container-scenarios.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ In most cases for this scenario, you will not need to migrate your existing appl
 
 ## Using third-party .NET libraries or NuGet packages not available for .NET Core
 
-Third-party libraries are quickly embracing the [.NET Standard](../../net-standard.md), which enables code sharing across all .NET flavors, including .NET Core. With the .NET Standard Library 2.0 and beyond the API surface compatibility across different frameworks has become significantly larger and in .NET Core 2.x applications can also directly reference existing .NET Framework libraries (see [compat shim](https://github.com/dotnet/standard/blob/master/docs/netstandard-20/README.md#net-framework-461-supporting-net-standard-20)).
+Third-party libraries are quickly embracing the [.NET Standard](../../net-standard.md), which enables code sharing across all .NET flavors, including .NET Core. With the .NET Standard Library 2.0 and beyond the API surface compatibility across different frameworks has become significantly larger and in .NET Core 2.x applications can also directly reference existing .NET Framework libraries (see [compat shim](https://github.com/dotnet/standard/blob/master/docs/planning/netstandard-2.0/README.md#net-framework-461-supporting-net-standard-20)).
```
Comment (rpetrusha) [inline docs/standard/microservices-architecture/net-core-net-framework-containers/net-framework-container-scenarios.md]:
```suggestion
Third-party libraries are quickly embracing the [.NET Standard](../../net-standard.md), which enables code sharing across all .NET flavors, including .NET Core. With the .NET Standard Library 2.0 and beyond the API surface compatibility across different frameworks has become significantly larger and in .NET Core 2.x applications can also directly reference existing .NET Framework libraries (see [.NET Framework 4.6.1 supporting .NET Standard 2.0](https://github.com/dotnet/standard/blob/master/docs/planning/netstandard-2.0/README.md#net-framework-461-supporting-net-standard-20)).
```

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks for fixing the broken link, @NextTurn. I thought that the link title might also be changed, since the content of the section doesn't reflect what most developers think of as a shim.

**Comment by mairaw** [comment]
@mvelosop FYI on a broken link from the e-book

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thank you, @NextTurn, for making the additional change. I'll merge your PR now.

**Comment by mvelosop** [comment]
Thank you @mairaw

**Pull request merged**